test_mcp.py

```
@@ -0,0 +1,72 @@
+"""
+import subprocess
+import json
+import sys
+
+def test_mcp_server():
+    """Testa o servidor MCP enviando requests via stdio."""
+    
+    print("🧪 Testando servidor MCP...\n")
+    
+    # Inicializar
+    init_request = {
```

Ruff (0.15.2)

[error] Local variable `init_request` is assigned to but never used

Remove assignment to unused variable `init_request`

(F841)